source/our_tools/tractoflow.rst

```diff
@@ -3,4 +3,220 @@
+            #SBATCH --cpus-per-task=32
+            #SBATCH --mem=0
+            #SBATCH --time=24:00:00
+
+            export NXF_CLUSTER_SEED=$(shuf -i 0-16777216 -n 1)
+
+            srun (copy your nextflow command from point 9) -with-mpi
+
+    #. Finally launch your sbatch! Yeah!
+
+        .. code-block:: bash
+
+            sbatch -A def-descotea cmd_my_tractoflow.sh
+
+            squeue -u USER  # To check if it has been launched
+
+            exit # To quit beluga
+
+        You will receive an email when your command is launched (if you provided it in step 11). If you want to supervise the results while it runs, you should find a file such as slurm-6635828.out. It is the equivalent of what would be printed in your terminal if you ran it directly. You can download it with scp or simply look at it using 'cat slurm-6635828.out'.
+
+        .. note:: It is normal to see some failed tasks. The way it works, many jobs are started at the same time, and it may cause some scheduling errors. As long as some jobs succeed, you can let it run.
+
+    #. We recommend doing something like this to save results, scripts and container
+
+        .. code-block:: bash
+
+            mkdir final_results
+            mkdir final_results/containers/
+            mkdir final_results/scripts
+            cp output_tractoflow/cmd_*.sh final_results/scripts/
+            cp output_tractoflow/beluga.conf final_results/scripts/
+            cp -rL output_tractoflow/results final_results/tractoflow
+            cp -rL qa-nf/results_QA final_results/qa-tractoflow
+
+            mv *img final_results/containers/
+            cp *.txt final_results/scripts/
+
+            # If you are sure to be done, uncomment following line
+            # rm -rf input_* output_*
+
+Checking the results
+********************
+
+When the job is finished, you can check the slurm output, as in step 12. The last part should look like:
+
+    .. code-block:: bash
+
+        Pipeline completed at: Thu Apr 09 20:45:08 EDT 2020
+        Execution status: OK
+        Execution duration: 5h 32m 57s
+        Completed at: 09-Apr-2020 20:45:09
+        Duration    : 5h 32m 59s
+        CPU hours   : 129.2 (2% cached, 0% failed)
+        Succeeded   : 286
+        Cached      : 98
+        Ignored     : 1
+        Failed      : 3
+
+In this example, you see 3 fails and 1 ignored. When tractoflow fails to preprocess a subject, it tries again up to 4 times, at after the last time, the subject is ignored and tractoflow continues with the rest of the pipeline. So here, the 3 fails and 1 ignore are for the same subject.
+
+To discover which subject caused a problem, you may check the report.html. Scrolldown to the task table, and look for subjects with 'failed' status (you can you the search bar).
+
+To discover the reason for fails, there is no easy answer. You might have to check each file individually, see if some files are corrupted or how their brain looks like.
+
+Using config files
```

Comment:
Please never modify the nextflow.config file. If you do that it is at your own risks. I HIGHLY recommend to use the options in command line instead modifying or creating a new nextflow.config. I prefer to remove this part.

Reply:
For HCP data, @arnaudbore  recommended me to use the config file in BrainData. You should see together what would be the best technique.

Reply:
@arnaudbore Why do you ask to modify the nextflow.config ? I think it is not a good idea. It is better to use the options in command line because if you want to use the same code in an other project the default are not the real default values

Reply:
Sorry guys I don't recall this one...

Reply:
I agree with @GuillaumeTh **IF** the way Tractoflow has run (command line/parameters) is explicit somewhere (code folder would be the best).

Reply:
Added a comment.